### DISCUSSION TITLE
Questions about Exodus file

### DISCUSSION THREAD

**Discussion by bosxered** [idaholab/moose#23754] [Q&A General] [answered]
Hello Moose experts,

I have some simple and basic questions about the exodus file output.
Here are my exodus output files.
![image](https://user-images.githubusercontent.com/79071957/225520137-7c99de35-6049-4095-8ff4-a7b183be09dc.png)
![image](https://user-images.githubusercontent.com/79071957/225520181-af1ecf49-82c8-4c53-8a73-98575cc1b5cc.png)

For some examples, exodus files have the suffix, such as -"s002", but sometimes don't (only one Exodus file, output.e).

1. What does this s002 or s003 means? How can I control this?
Also, I want to know why output.e file has 14MB data, some have 696KB data, and the output.e-148 has 40MB data.

2. Are they split form of one Exodus file? If not, is there any way to split a large Exodus file into several small exodus files?
For example,
from
one Exodus file: output.e (including results from timestep 1 to timestep 100)
to
10 Exodus files: output.e (timestep 1 to 10), output.e-2 (timestep 11 to 20), output.e-3 (timestep 21 to 30), ......

Thank you for reading my bothering questions.

**Comment by GiudGiud** [comment, marked as answer]
Hello

The -s00xxx file extension comes from the mesh being modified and the exodus output not being able to use the same file.

For example if you modify the number of elements, with adaptivity, or if you move the mesh some way (using MultiApp move or reset), then this appears.

The amount of data depends on the size of the mesh and the number of time steps mainly. I suspect that's what s happening for that 40MB file is that you have additional input on FINAL ? You should be able to see the difference in content from opening the files in paraview. 

For 2, you can create multiple subblocks within Outputs and them all be Exodus. Then you can use the "timing of ouptuts parameters to stagger them. For example:
```
[Ouputs]
  [exo_1]
    type = Exodus
    start_time = 3
    end_time = 10
  []
  [exo_2]
    type = Exodus
    start_time = 10
    end_time = 100
  []
```
https://mooseframework.inl.gov/source/outputs/Exodus.html

**Reply by bosxered** [reply]
Thanks for your answer!

Actually, I don't think I fully understand your answer.
So, I want to see changes in the system from t=0 to t=END by ParaView.

How can I do this?

Is it enough to watch output.e-sFINAL? I don't think it's right according to your answer.

**Reply by GiudGiud** [reply]
I m not sure if the final one has everything, I would think not. 
What do you see?

You can always open multiple files in paraview

**Reply by bosxered** [reply]
Okay. Thanks for your kindness. Have a good day!
